app/src/main/res/values/strings.xml

```diff
@@ -113,6 +113,7 @@
     <string name="never">never</string>
     <string name="last_update">Last update</string>
     <string name="update_ad_blocker_list">Update ad-blocker list</string>
+    <scring name="toggle_keep_screen_on">Toogle keep screen on</string>
```

Comment:
`scring` ?

I guess you mean `string` here.

This will not even compile properly, I guess you should try to compile and test your changes before opening pull requests :)

Reply:
Definitely :)

I'm new to kotlin/android programming (actually just for this MR), but am happy to read through/set up everything necessary to help out.

Would it be ok for you all, if I add a *Contributing* section to the README after this MR is finished? To note for newbies like me what needs to be set up to compile it, to run linter rules/automated tests (if any) and how to test the app locally/manually?

Reply:
No problem! And good idea with contributing section. I've been too lazy to do so until now))